src/main/java/com/bean00/Response.java

```diff
@@ -1,9 +1,34 @@
 package com.bean00;
 
+import java.util.HashMap;
+
 public class Response {
+    private String version = "HTTP/1.1";
```

Comment:
can you try making this immutable?

Reply:
I made it a constant using `private static final`.